CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 2.8.7)
```

Comment:
Is this correct?

`target_sources()` is used below, and requires CMake >= 3.1.0, I think:
- https://cmake.org/cmake/help/v3.1/release/3.1.0.html?highlight=target_sources#commands